OrbitQt/orbitmainwindow.cpp

```diff
@@ -128,6 +128,16 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App,
       [this] { ui->RightTabWidget->setCurrentWidget(ui->LiveTab); });
   GOrbitApp->AddDisassemblyCallback(
       [this](const std::string& disassembly) { OpenDisassembly(disassembly); });
+  GOrbitApp->AddErrorMessageCallback(
+      [this](const std::string& title, const std::string& text) {
+        QMessageBox::critical(this, QString::fromStdString(title),
+                              QString::fromStdString(text));
+      });
+  GOrbitApp->AddInfoMessageCallback(
```

Comment:
are they singular callbacks? maybe replace add with set and remove loops?

Reply:
The vast majority if not all of them are, yes. I was following the pattern of the previous code. I'll switch to single callbacks.

Reply:
Done.